src/components/logs/logsWebview.ts

```diff
@@ -80,6 +82,12 @@ export class LogsPanel extends WebPanel {
                     });
                     // TODO: need to apply filters here!
                     elt.appendChild(document.createTextNode(message.text));
+
+                    // handle auto-scroll on/off
+                    var checkBox = document.getElementById("autoscroll");
+                    if (checkBox.checked == true){
+                        document.getElementById('bottom').scrollIntoView();
```

Comment:
This element doesn't seem to be present in the HTML.  I found if I added a `<p id='bottom'></p>` after the `<code>...</code>` around line 59-62 then it worked, but that was only really simple testing...

Reply:
Sorry about missing that in my first pass, but I have added it back in.